NOpenType/GlyphPathBuilderBase.cs

```diff
@@ -103,43 +104,27 @@ void RenderGlyph(ushort[] contours, FtPoint[] ftpoints, int[] flags)
                         switch (controlPointCount)
                         {
                             case 0:
-                                {   //bit 1 set=> off curve, this is a control point
-                                    //if this is a 2nd order or 3rd order control point
-                                    if (((vtag >> 1) & 0x1) != 0)
-                                    {
-                                        //printf("[%d] bzc3rd,  x: %d,y:%d \n", mm, vpoint.x, vpoint.y);
-                                        thirdControlPoint = new FtPointD(vpoint);
-                                    }
-                                    else
-                                    {
-                                        //printf("[%d] bzc2nd,  x: %d,y:%d \n", mm, vpoint.x, vpoint.y);
-                                        secondControlPoint = new FtPointD(vpoint);
-                                    }
+                                {
+                                    secondControlPoint = new FtPointD(vpoint_x, vpoint_y);
                                 }
```

Comment:
What happened here?? Please do one thing per pull request, otherwise this will take 100 years...